Apps/WebClient/src/ClientApp/src/components/calendar/calendar.vue

```diff
@@ -4,6 +4,7 @@
         <CalendarHeader
             :current-date.sync="currentDate"
             :title-format="titleFormat"
+            :availiable-months="availiableMonths"
```

Comment:
typo: availiable instead of available in many places. Mbe replace all in the solution.

Reply:
updated, throughout